src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestClient.java

```diff
@@ -22,21 +20,10 @@
    */
   SnowflakeStreamingIngestChannel openChannel(OpenChannelRequest request);
 
-  /**
-   * Close all the channels in the client, which will make sure all the data in all channels is
-   * committed before closing
-   *
-   * @return a completable future which will be completed when all the channels are closed
-   */
-  CompletableFuture<Void> close();
-
   /**
    * Get the client name
    *
    * @return the client name
    */
   String getName();
-
-  /** @return a boolean to indicate whether the client is closed */
-  boolean isClosed();
```

Comment:
I think we should keep isClosed() just so as to not get client is not initialised error while calling open channel. 

Today I use isClosed() to do that.